ush/generate_FV3LAM_wflow.sh

```diff
@@ -402,7 +402,7 @@ file (template_xml_fp):
 #
 # METPlus-specific information
 #
-  'ccpa_obs_dir': ${CCPA_OBS_DIR}
+  'ccpa_obs_dir': $( eval echo ${CCPA_OBS_DIR} )
   'mrms_obs_dir': ${MRMS_OBS_DIR}
   'ndas_obs_dir': ${NDAS_OBS_DIR}
 # Move the variable NET to somewhere more appropriate.
```

Comment:
Does this change only apply to the CCPA directory and not MRMS or NDAS?

Reply:
Yeah, I think I should apply to the other two as well.  Will fix.  But it only matters if the `GET_OBS_...` tasks are activated, which they are not in all our experiments.

Reply:
Yep, good point.  We'll have everything staged.